paddle/fluid/platform/macros.h

```diff
@@ -25,6 +25,6 @@ limitations under the License. */
   classname& operator=(classname&&) = delete
 #endif
 
-#if defined(__FLT_MAX__)
-#define FLT_MAX __FLT_MAX__
-#endif  // __FLT_MAX__
+//#if defined(__FLT_MAX__)
```

Comment:
same above, also need to use `#ifdef PADDLE_WITH_MUSL`, avoid introducing other problems

Reply:
ready in 2e12aebf246b69294ff57ee797105b43ab288b0e